scripts/deploy-rpc-playbooks.sh

```diff
@@ -39,6 +39,12 @@ if [[ "${DEPLOY_ELK}" == "yes" ]]; then
 fi
 
 # Download the latest release of rpc-maas
+# TODO(odyssey4me):
+# Remove this once rpc-gating no longer tries
+# to run rpc-maas as its own thing and instead
+# just uses deploy.sh end-to-end. This line
+# should not be necessary as setup-maas.yml
+# below includes maas-get.yml
 run_ansible maas-get.yml
```

Comment:
Could you also add that maas-get is needed even if DEPLOY_MAAS is not "yes"?
For example, it will be used for the influx db integration.